.sass-lint.yml

```diff
@@ -0,0 +1,47 @@
+# Linter Options
+options:
+  merge-default-rules: true
+
+# Rule Configuration
+rules:
+  class-name-format:
+  - 1
+  -
+    convention: 'hyphenatedlowercase'
+  mixin-name-format:
+    - 1
+    -
+      convention: 'camelcase'
+  variable-name-format:
+    - 1
+    -
+      convention: 'camelcase'
+  function-name-format:
+      - 1
+      -
+        convention: 'camelcase'
+  quotes:
+      - 1
+      -
+        style: 'double'
+  indentation:
+      - 1
+      -
+        size: 4
+  hex-notation:
+      - 1
+      -
+        style: 'lowercase'
+  hex-length:
+      - 1
+      -
+        style: 'long'
+  single-line-per-selector:
+      - 0
+  placeholder-in-extend:
+      - 0
+  empty-line-between-blocks:
+      - 1
+      -
+        include: true
+        allow-single-line-rulesets: false
```

Comment:
- Indent this file by tabs
- Please add the mentioned *Property sort order* with the http://rhodesmill.org/brandon/2011/concentric-css/ option
- Add missing blank line on the end of the file
- Use double quotes instead of single quotes